packages/make-styles/package.json

```diff
@@ -27,15 +27,14 @@
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.0.1",
     "@fluentui/scripts": "^1.0.0",
-    "@fluentui/test-utilities": "^8.0.1",
-    "@types/stylis": "^4.0.0"
+    "@fluentui/test-utilities": "^8.0.1"
   },
   "dependencies": {
     "@emotion/hash": "^0.8.0",
     "csstype": "^2.6.7",
     "inline-style-expand-shorthand": "^1.2.0",
     "rtl-css-js": "^1.14.0",
-    "stylis": "^4.0.6",
+    "stylis": "^4.0.7",
```

Comment:
didn't realise that newer version has removed typings and move it to 3rd party ....

- I bumped to same version that is being installed in our monorepo root(single version policy/ shouldn't affect consumers (minor bump))
- this satisfies syncpack for now. what needs to be done is to (get rid of syncpack or add functionality that will allow us to satisfy strict versioning in root/non strict in package dep, although I don't recommend this approach)

Reply:
Related thoughts (nothing that needs to be addressed in this PR):

> - this satisfies syncpack for now. what needs to be done is to (get rid of syncpack or add functionality that will allow us to satisfy strict versioning in root/non strict in package dep, although I don't recommend this approach)

The developer of syncpack has been pretty responsive to feature requests in the past (they added versionGroups support for us), so it might be worth asking about this feature too if we can figure out a more specific proposal.

Another possibility is removing the root package.json from the `syncpack.source` array, though that has its own risks which we'd probably need some additional tooling to address.